### DISCUSSION TITLE
Is the face detection algorithm run in the cloud or locally on the server?

### DISCUSSION THREAD

**Discussion by ark-** [photoview/photoview#246] [General]
If face detection is run locally and no images are ever sent to any web services then I believe this should be stated in the readme as that could be considered a useful feature for people moving from, e.g. Google Photos.

**Comment by viktorstrate** [comment]
Hi @ark- great question!
The face recognition is done 100% locally and the face descriptors are saved in the database, if the database is hosted locally as well (it most likely is), then nothing leaves the server!

It's very subtle but the [website](https://photoview.github.io/) hints it, I think it would be a good idea to emphasize it though, as it might be an important topic for many.
<img width="331" alt="Screenshot 2021-03-08 at 19 56 41" src="https://user-images.githubusercontent.com/4233458/110367775-68d93c80-8048-11eb-9a9b-16be89b2cd6a.png">

**Reply by ark-** [reply]
Ah yes, I did not check the website. Went directly to github - my bad. Thanks for your response :-)
